src/vmm/src/config/mod.rs

```diff
@@ -140,6 +140,8 @@ pub struct KernelConfig {
     pub path: PathBuf,
     /// Start address for high memory.
     pub himem_start: u64,
+    /// Optional path to pipe the input and output of the gues OS
+    pub serial_console_path: Option<String>,
```

Comment:
I don't think this is conceptually a guest kernel configuration. Since it's an option for how to operate one of the VMM's devices, I'd rather see it in a new `SerialConfig` (or something...I'm bad with names) struct.